### ISSUE TITLE
EmailTemplate.name should be indexed

### ISSUE THREAD

**Issue by podloucky-init** [ui/django-post_office#250]
Hi there,
First of all: thank you for providing django-post-office. We are currently evaluating it as part of an internal mail-distribution service and so far it is looking good!

Templates are usually looked up by name (using `utils.get_email_template()`) but the field `EmailTemplate.name` is not indexed which may lead to bad performance.
I think it would make sense to index that field even when using a cache.

Cheers,
Konrad

**Comment by selwin** [comment]
Yeah, this is definitely an oversight. Could you please open a PR for this?
